### PR TITLE
Xo

### PR DESCRIPTION
**Pull request by scopatz**
ye ole' text editor.

### CODE REVIEW

**Comment by scopatz** [comment]
Whoohoo it works!

File: recipes/xo/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,36 @@
+{% set version = "0.1.11" %}
+
+package:
+  name: xo
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/scopatz/xo.git
+  git_tag: 8f486a2709d376d00f98c09a86179bce2a99099a
```
Comment (pelson) [inline recipes/xo/meta.yaml]: 😞  - are you likely to tag ```v0.1.11```? If so, I'd encourage you to download the prepared ```.tar.gz``` linked from there. If not, maybe you could use https://github.com/scopatz/xo/archive/8f486a2709d376d00f98c09a86179bce2a99099a.tar.gz.

That said, given the size of the repo, it isn't a big deal either way.

File: recipes/xo/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    - python >=3.4
+    - setuptools
+
+test:
+  commands:
+    - xo --help
+
+about:
+  home: http://exofrills.org/
+  license: WTFPL v2
```
Comment (pelson) [inline recipes/xo/meta.yaml]: My first exposure to WTFPL. 😆

File: recipes/xo/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+  number: 0
+  skip: true  # [win]
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  run:
+    - python >=3.4
+    - urwid >=1.1.1
+    - pygments >=1.6
+  build:
```
Comment (pelson) [inline recipes/xo/meta.yaml]: Apparently the example did this too, but would you mind swapping the order of build and run?

File: recipes/xo/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+  git_tag: 8f486a2709d376d00f98c09a86179bce2a99099a
+
+build:
+  number: 0
+  skip: true  # [win]
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  run:
+    - python >=3.4
```
Comment (pelson) [inline recipes/xo/meta.yaml]: We need to be super careful here - conda does not do what you (or I) would expect. Sure, conda will decide to build with a ```python >=3.4```, but the Python that is used to ```build``` the package is *not* necessarily the python that is used to ```run``` the package.
Adding another line which states ```python x.x``` is necessary (I think both in the run and in build sections, but you can test that by looking at the build string and observing a ```py34```/```py35``` suffix)

**Comment by pelson** [comment]
Couple of comments, but nothing major. 👍

File: recipes/xo/meta.yaml (hunk 2 above)
Comment (scopatz) [inline recipes/xo/meta.yaml]: Nice :)

**Comment by scopatz** [comment]
Thanks for the review @pelson! Is this more like what you where thinking?

File: recipes/xo/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,38 @@
+{% set version = "0.1.11" %}
+
+package:
+  name: xo
+  version: {{ version }}
+
+source:
+  fn: xo.tar.gz
+  url: https://github.com/scopatz/xo/archive/8f486a2709d376d00f98c09a86179bce2a99099a.tar.gz
```
Comment (pelson) [inline recipes/xo/meta.yaml]: I guess we would typically also add a sha256 to assert that somebody malicious hasn't man-in-the-middled you, but it isn't a deal breaker. If you also put the version in the filename, that would stop conda build from thinking it already has the file in its cache. i.e. this section becomes:

```
source:
  fn: xo-{{ version }}.tar.gz

  url: https://github.com/scopatz/xo/archive/8f486a2709d376d00f98c09a86179bce2a99099a.tar.gz
```

Comment (pelson) [inline recipes/xo/meta.yaml]: Sorry, tapped ctrl+enter by accident. I edited my last message, hope you got all of that.

**Comment by pelson** [comment]
Pretty much, yes @scopatz

**Comment by pelson** [comment]
👍

**Comment by scopatz** [comment]
Ok, how about now? :)

File: recipes/xo/meta.yaml [hunk 6]
```diff
@@ -0,0 +1,41 @@
+{% set version = "0.1.11" %}
+# obtained sha256 with "shasum -a -b" command
+{% set sha256 = "bab75f582326bb0db4e49a991ec075908b73fab91e0110482304f21ae5e34cd2" %}
+
+package:
+  name: xo
+  version: {{ version }}
+
+source:
+  fn: xo-{{ sha256 }}.tar.gz
```
Comment (pelson) [inline recipes/xo/meta.yaml]: How cunning! 😄

**Comment by pelson** [comment]
Sure is 👍

File: recipes/xo/meta.yaml (hunk 6 above)
Comment (scopatz) [inline recipes/xo/meta.yaml]: hahah

**Pull request merged**